### DISCUSSION TITLE
Which Language Should Be Next Added to the Voice Cloning in Wunjo AI?

### DISCUSSION THREAD

**Discussion by wladradchenko** [wladradchenko/wunjo.wladradchenko.ru#31] [Ideas]
**Greetings everyone,**

Expanding the language options for voice cloning within Wunjo AI application is a significant step forward. I've outlined the complexities of this task in detail [here](https://github.com/wladradchenko/wunjo.wladradchenko.ru/discussions/2#discussioncomment-7065675). However, recognizing that not everyone may have the experience or resources for model training, I'm initiating this discussion to gather your input.

Adding a new language requires a meticulous process, including incorporating language-specific features, sourcing appropriate training datasets, and conducting thorough training and testing. To determine the priority for language implementation, I'll start by listing languages, but please feel free to contribute any additional languages I might have overlooked.

The priority of each language will be determined by the number of likes garnered in individual comments. To ensure clarity and fairness in the voting process, kindly submit one language per message.

**For easy searching, I have marked the voting languages with a hashtag #vote.**

Additionally, if you have access to datasets suitable for training in your desired language, sharing those links here would be immensely helpful.

Moreover, the application supports training a TTS (Text-to-Speech) model for speech synthesis. For those interested, I've created a tutorial video outlining the process [here](https://www.youtube.com/watch?v=KY7mdB9rXbw).

Let's work together to enhance the available of voice cloning application, Wunjo AI for international using. Your input and participation are greatly appreciated.

**Thank you for contributing!**

**Comment by wladradchenko** [comment]
Spanish Language #vote

**Comment by wladradchenko** [comment]
Czech Language #vote

**Comment by wladradchenko** [comment]
German Language #vote

**Reply by alexgold1973** [reply]
немецкий было бы круто

**Comment by wladradchenko** [comment]
Japan Language #vote

**Comment by wladradchenko** [comment]
French Language #vote

**Comment by wladradchenko** [comment]
Italian Language #vote

**Comment by wladradchenko** [comment]
Norwegian Language #vote

**Comment by wrappan** [comment]
Здравствуйте.
Спасибо за проделанный труд.

**Comment by thomaskarado** [comment]
Portuguese (Brazil), PLEASE!!!

**Comment by wladradchenko** [comment]
Voice cloning now supports any language, the discussion topic has run its course.
